awswrangler/data_quality/_get.py

```diff
@@ -9,28 +9,39 @@
 
 
 def get_ruleset(
-    name: str,
+    name: Union[str, List[str]],
     boto3_session: Optional[boto3.Session] = None,
 ) -> pd.DataFrame:
     """Get a Data Quality ruleset.
 
     Parameters
     ----------
-    name : str
-        Ruleset name.
+    name : str or list[str]
+        Ruleset name or list of names.
     boto3_session : boto3.Session, optional
         Boto3 Session. If none, the default boto3 session is used.
 
     Returns
     -------
     pd.DataFrame
-        Data frame with ruleset details.
+        Data frame with ruleset(s) details.
 
     Examples
     --------
+    Get single ruleset
     >>> import awswrangler as wr
 
     >>> df_ruleset = wr.data_quality.get_ruleset(name="my_ruleset")
+
+    Get multiple rulesets. A column with the ruleset name is added to the data frame
+    >>> df_rulesets = wr.data_quality.get_ruleset(name=["ruleset_1", "ruleset_2"])
     """
-    rules = cast(str, _get_ruleset(ruleset_name=name, boto3_session=boto3_session)["Ruleset"])
-    return _rules_to_df(rules=rules)
+    ruleset_names: List[str] = name if isinstance(name, list) else [name]
+    dfs: List[pd.DataFrame] = []
+    for ruleset_name in ruleset_names:
+        rules = cast(str, _get_ruleset(ruleset_name=ruleset_name, boto3_session=boto3_session)["Ruleset"])
+        df = _rules_to_df(rules=rules)
+        if len(ruleset_names) > 1:
```

Comment:
I think a column with the ruleset name should only be added if there is multiple ones. But let me know if you disagree

Reply:
What's the major downside to adding the ruleset name always? I like the idea of the structure of the DataFrame to be consistent (i.e. it should either always have `ruleset`).